Dockerfile

```diff
@@ -3,15 +3,21 @@
 # VERSION	0.1
 
 FROM debian:jessie-slim
-MAINTAINER Richard North <rich.north@gmail.com>
 
 LABEL Description="This image can be used to create a sidekick container for recording videos of VNC sessions hosted in other containers"
+LABEL maintainer="Richard North <rich.north@gmail.com>"
 
 RUN apt-get update && apt-get install -y \
   python-pip python-dev \
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* \
+  && pip install vnc2flv \
```

Comment:
Is this used?

Reply:
Yes It's used (It was in the previous version), I have just grouped these commands to reduce image size, the library [vnc2flv](https://pypi.org/project/vnc2flv/) is used (for now) to record Selenium test.